language_service/src/name_locator.rs

```diff
@@ -5,19 +5,24 @@ use std::mem::replace;
 use std::rc::Rc;
 
 use crate::compilation::Compilation;
-use qsc::ast::visit::{walk_expr, walk_namespace, walk_pat, walk_ty, walk_ty_def, Visitor};
+use qsc::ast::visit::{
+    walk_attr, walk_expr, walk_namespace, walk_pat, walk_ty, walk_ty_def, Visitor,
+};
 use qsc::ast::{FieldAccess, Idents, PathKind};
 use qsc::display::Lookup;
 use qsc::{ast, hir, resolve};
 
 #[allow(unused_variables)]
```

Comment:
Could we remove the `#[allow(unused_variables)]` attribute here? There doesn't seem to be any unused variables.

Reply:
done